src/main/java/edu/uc/javahotkey/dao/IProjectRetrofitDAO.java

```diff
@@ -0,0 +1,18 @@
+package edu.uc.javahotkey.dao;
+
+import edu.uc.javahotkey.dto.KeyMap;
+import edu.uc.javahotkey.dto.Project;
+import org.aspectj.weaver.ast.Call;
+import org.springframework.data.jpa.repository.Query;
+import retrofit2.http.GET;
+
+import java.util.List;
+
+public interface IProjectRetrofitDAO {
+
+    //TODO update method with actual JSON endpoint
+
+    @GET("/")
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.